src/cfnlint/__init__.py

```diff
@@ -346,10 +346,13 @@ def create_from_directory(cls, rulesdir):
 class RuleMatch(object):
     """Rules Error"""
 
-    def __init__(self, path, message):
+    def __init__(self, path, message, **kwargs):
         """Init"""
         self.path = path
+        self.path_string = '/'.join(map(str, path))
```

Comment:
Whats the benefit to this one?

Reply:
No idea. I didn't add it.

Reply:
OH, yes I did add it. If... i remember correctly... the path is a list... and turning it into a string means it can be added to an attribute.